src/SDK/MonitoringApplianceEmulator/ViewModels/TracesControlViewModel.cs

```diff
@@ -0,0 +1,272 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="TracesControlViewModel.cs" company="Microsoft Corporation">
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// </copyright>
+//-----------------------------------------------------------------------
+
+namespace Microsoft.Azure.Monitoring.SmartDetectors.MonitoringApplianceEmulator.ViewModels
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Collections.ObjectModel;
+    using System.ComponentModel;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Monitoring.SmartDetectors.MonitoringApplianceEmulator.Models;
+    using Microsoft.Azure.Monitoring.SmartDetectors.MonitoringApplianceEmulator.Trace;
+    using Unity.Attributes;
+
+    /// <summary>
+    /// The view model for the traces control.
+    /// </summary>
+    public class TracesControlViewModel : ObservableObject
```

Comment:
>TracesControlViewModel [](start = 17, length = 22)

no need for UTs for this one?

Reply:
good point. added

---
In reply to: [231572528](https://github.com/Azure/SmartDetectorsMonitoringAppliance/pull/13#discussion_r231572528) [](ancestors = 231572528)